patches.python/0001-libvirtaio-add-more-debug-logging.patch

```diff
@@ -0,0 +1,90 @@
+From f95c94d30296f7e9b897dcb1332db4fad2b9db57 Mon Sep 17 00:00:00 2001
+From: Wojtek Porczyk <woju@invisiblethingslab.com>
+Date: Thu, 24 Aug 2017 21:22:49 +0200
+Subject: [PATCH 1/6] libvirtaio: add more debug logging
+Organization: Invisible Things Lab
+Cc: Wojtek Porczyk <woju@invisiblethingslab.com>
+
+This logging is helpful for tracing problems with unclosed connections
+and leaking file descriptors.
+
+Signed-off-by: Wojtek Porczyk <woju@invisiblethingslab.com
```

Comment:
You forgot `>` here. I recommend `git format-patch -s`, or `git config format.signOff true`